website/docs/r/team_notification_configuration.html.markdown

```diff
@@ -0,0 +1,132 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_team_notification_configuration"
+description: |-
+  Manages team notifications configurations.
+---
+
+# tfe_team_notification_configuration
+
+HCP Terraform can be configured to send notifications to a team for certain events.
+Team notification configurations allow you to specify a URL, destination type, and what events will trigger the notification.
+Each team can have up to 20 notification configurations, and they apply to configured events for all workspaces that the configured team has access to.
+
+## Example Usage
+
+Basic usage:
+
+```hcl
+resource "tfe_organization" "test" {
+  name  = "my-org-name"
+  email = "admin@company.com"
+}
+
+resource "tfe_team" "test" {
+  name         = "my-team-name"
+  organization = tfe_organization.test.id
+}
+
+resource "tfe_team_notification_configuration" "test" {
+  name             = "my-test-notification-configuration"
+  enabled          = true
+  destination_type = "generic"
+  triggers         = ["change_request:created"]
+  url              = "https://example.com"
+  team_id          = tfe_team.test.id
+}
+```
+
+With `destination_type` of `email`:
+
+```hcl
+resource "tfe_organization" "test" {
+  name  = "my-org-name"
+  email = "admin@company.com"
+}
+
+resource "tfe_team" "test" {
+  name         = "my-team-name"
+  organization = tfe_organization.test.id
+}
+
+resource "tfe_organization_membership" "test" {
+  organization = "my-org-name"
+  email        = "test.member@company.com"
+}
+
+resource "tfe_team_notification_configuration" "test" {
+  name             = "my-test-email-notification-configuration"
+  enabled          = true
+  destination_type = "email"
+  email_user_ids   = [tfe_organization_membership.test.user_id]
```

Comment:
this supposedly doesn't work on non-TFE

Reply:
I thought you mentioned that emailing specific users was only on TFE, I may have misunderstood?

Reply:
I updated the example add the user to the team, not sure if that's the part that was failing for you or if it was something else.

Reply:
the `email_addresses` will only work for TFE (there's an example for that one below this as well). but `email_user_ids` should work fine for both.